controllers/authController.js

```diff
@@ -64,3 +75,76 @@ exports.loginUser = (req, res) => {
         return res.status(200).json({message: "Successfully logged in", token})
   })
 }
+
+//This function is responsible to send email to a particular user with the token for password reset
+exports.forgotPassword = (req, res) => {
+  User.findOne({email: req.body.email} , (err, foundUser) => {
+    if (err || !foundUser) {
+      return res.status(400).json({message: "User with this email does not exist."})
+    }
+    
+    let token = createToken({_id: user._id}, process.env.RESET_PASSWORD_KEY, {expiresIn: '20m'});
```

Comment:
using a jwt is not the best. You can create a schema for this token and generate random strings as token. Something like:
user: Schema.Types.ObjectId
token: String
expiresAt: String